File: gitea_repos_master/getcontact/getcontact-mobile-api/src/exceptions/exceptions.py
class MyError(Exception):
    def __init__(self, text=''):
        self.message = text


class AccountBlocked(MyError):
    pass


class AccountLocked(MyError):
    pass


class CaptchaDetected(MyError):
    pass


class LimitError(MyError):
    pass


class UnknownError(MyError):
    pass


class NoDataError(MyError):
    pass

File: gitea_repos_master/whatsapp/whatsapp-baileys-api/src/logic/exceptions/exceptions.ts
class MyError extends Error {
  constructor(message: string) {
    super(message);
  }
}
export class ErrorNoReturnToQueue extends MyError {}
export class ErrorReturnToQueue extends MyError {}
export class AccountBlocked extends ErrorReturnToQueue {}
export class AccountLocked extends ErrorReturnToQueue {}

export class NoDataError extends ErrorNoReturnToQueue {}
export class LimitError extends ErrorReturnToQueue {}
export class TimeoutError extends ErrorReturnToQueue {}
export class ConnectionError extends ErrorReturnToQueue {}
export class UnknownError extends ErrorReturnToQueue {}
